docs/content/users/configuration/config.md

```diff
@@ -118,7 +118,7 @@ The type and version of the database engine the project should use.
 
 | Type | Default       | Usage
 | -- |---------------| --
-| :octicons-file-directory-16: project | MariaDB 10.11 | Can be MariaDB 5.5–10.8, 10.11, or 11.4, MySQL 5.5–8.0, or PostgreSQL 9–15.<br>See [Database Server Types](../extend/database-types.md) for examples and caveats.
+| :octicons-file-directory-16: project | MariaDB 10.11 | Can be MariaDB 5.5–10.8, 10.11, 11.4, 11.8 and MySQL 5.5–8.0, or PostgreSQL 9–15.<br>See [Database Server Types](../extend/database-types.md) for examples and caveats.
```

Comment:
10.7 and 10.8 are EOL short term support releases. Just curious how you are treating eol 11.6 versions differently from 10.7?

Or currently STS GA version like 11.7 ?

Reply:
10.7 and 10.8 were done before we started the "LTS only" policy.